go/service/git.go

```diff
@@ -243,6 +247,79 @@ func (h *GitHandler) DeleteTeamRepo(ctx context.Context, arg keybase1.DeleteTeam
 	return git.HumanizeGitErrors(ctx, h.G(), err)
 }
 
+func (h *GitHandler) GcPersonalRepo(ctx context.Context, arg keybase1.GcPersonalRepoArg) (err error) {
+	ctx = libkb.WithLogTag(ctx, "GIT")
+	defer h.G().CTraceTimed(ctx, "git:GCPersonalRepo",
+		func() error { return err })()
+
+	client, err := h.kbfsClient()
+	if err != nil {
+		return err
+	}
+	folder := keybase1.Folder{
+		Name:       h.G().Env.GetUsername().String(),
+		FolderType: keybase1.FolderType_PRIVATE,
+		Private:    true,
+	}
+	options := keybase1.GcOptions{}
+	if !arg.Force {
+		options.MaxLooseRefs = gitDefaultMaxLooseRefs
```

Comment:
Just confirming, zero in this field means "definitely always gc"?

Reply:
Yep!